examples/hosts/iframe-host/src/outer.ts

```diff
@@ -71,31 +67,32 @@ async function getFluidObjectAndRender(baseHost: BaseHost, url: string, div: HTM
 export async function loadDiv(divId: string) {
     const div = document.getElementById(divId) as HTMLDivElement;
 
-    const urlResolver = getTinyliciousResolver();
+    const urlResolver = new InsecureTinyliciousUrlResolver();
 
     const documentServiceFactory = new RouterliciousDocumentServiceFactory();
 
-    const pkg: IFluidCodeDetails = {
-        package: "@fluid-example/todo@^0.15.0",
-        config: {
-            "@fluid-example:cdn": "https://pragueauspkn.azureedge.net",
-        },
+    // Dummy code details that won't really get used
+    const codeDetails: IFluidCodeDetails = {
+        package: "@fluid-example/todo",
+        config: { },
     };
 
-    const baseHost = new BaseHost(
-        {
-            codeResolver: new SemVerCdnCodeResolver(),
-            documentServiceFactory,
-            urlResolver,
-            options: {},
-        });
+    const module = { fluidExport: TodoContainer };
```

Comment:
This stuff can probably be refactored into another IFrameHost class similar to base-host (base-host expects to use the WebCodeLoader internally which isn't how the data object is being loaded here)